### PR TITLE
240 add autoseg project page

### LINKED ISSUE #240: Add AutoSeg Project Page
Create a project page for the AutoSegCT Aide app.

### PR DESCRIPTION
**Pull request by AnilMistry**
Resolves #240

### CODE REVIEW

File: _projects/autosegct.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+<b>Clinical lead</b>: Christopher Thomas, Teresa Guerrero Urbano
+<br>
+<br> **Rationale:**
+
+AI auto-contouring tools have improved dramatically in the last few years, and all major treatment planning system vendors are investing in this technology along with independent start-ups and NHS Trusts [NICE, 2023].
+
+GSTT are in a strong position to develop our own auto-contouring pipeline using the AI expertise within RT physics, clinical oncology, and the Clinical Scientific Computing (CSC) team, strong links with King’s College London (KCL) academics and outputs from three KCL PhD students creating AI auto-contouring models.
+
+Segmentation models have been created as output of the extensive research activity generated from Guys Cancer centre. This application will integrate the following models for evaluation in routine clinical care.
```
Comment (heyhaleema) [inline _projects/autosegct.md]:
```suggestion
Segmentation models have been created as output of the extensive research activity generated from Guy's Cancer Centre. This application will integrate the following models for evaluation in routine clinical care.
```

**Review by heyhaleema — CHANGES_REQUESTED** [review]
@AnilMistry LGTM apart from:

- I get this error:
   image
- One minor typo on Line 26 i.e., missing an apostrophe 👍 There's also some formatting quirks I'll post in a separate comment, but I don't know if this necessarily prevents approval/merge if we're just focusing on content for now.

**Comment by heyhaleema** [comment]
@AnilMistry A few formatting quirks:

- The (beautiful) logo seems to be cut off on the project card 🥲
- I wasn't sure but are tables and citations formatted as you expect? I feel like this might be Markdown vs. HTML issue...

Also, a general question for the group about projects: what's the standard for CSC leads e.g., first name only vs. full name, co-leads, etc.? @dangerdika @sophie22  @Beatrix-Fletcher @alexandertdeng

**Comment by dangerdika** [comment]
> @AnilMistry A few formatting quirks:
> 
>     * The (beautiful) logo seems to be cut off on the project card 🥲
> 
>     * I wasn't sure but are tables and citations formatted as you expect? I feel like this might be Markdown vs. HTML issue...
> 
> 
> Also, a general question for the group about projects: what's the standard for CSC leads e.g., first name only vs. full name, co-leads, etc.? @dangerdika @sophie22 @Beatrix-Fletcher @alexandertdeng

So far the standard has been first name only, we may need to revise this issue if we end up with repeated names in the team but for now I'd say just first name is enough

**Comment by dangerdika** [comment]
Also @AnilMistry I dont think you need to expand on the clinical scientific computing (CSC), I think you can just say 'CSC' without elaboration; but please expand the RT physics into 'radiotherapy physics'

File: _projects/autosegct.md (hunk 1 above)
Comment (AnilMistry) [reply _projects/autosegct.md]: Resolved

File: _projects/autosegct.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+---
+layout: project_page
+title: Radiotherapy Auto Segmentation
+status: Evaluating
+image: /assets/img/projects/autosegct_logo.png
+summary: Single application for all in-house auto-segmentation contours
+csc-lead: <a href="/team_member/Anil.html">Anil Mistry</a> <a href="/team_member/Tom.html">Tom Roberts</a>
```
Comment (heyhaleema) [inline _projects/autosegct.md]:
```suggestion
csc-lead: <a href="/team_member/Anil.html">Anil</a> <a href="/team_member/Tom.html">Tom</a>
```

Suggested edit in keeping with [current standard](https://github.com/GSTT-CSC/gstt-csc.github.io/pull/353#issuecomment-1983713501).

File: _projects/autosegct.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+2. **Prostate Model** - _Maram Alquarni_ 
+* Prostate, Rectum, Penile Bulb, Bladder, Femoral Head Left, Femoral Head Right 
+
+
+3.  **BowelSpace  model** - _Christopher Thomas_  
+* Bowel space             
+
+
+4.  **Head&Neck Model** -  _Tom Youngs_
```
Comment (heyhaleema) [inline _projects/autosegct.md]:
``` suggestion
4.  **Head&Neck Model** -  _Tom Young_  
```
Is this a typo?

**Review by heyhaleema — APPROVED** [review]
LGTM 🔥

**Pull request merged**